### PR TITLE
Lsanabria/git lfs test

### PR DESCRIPTION
**Pull request by SanabriaRusso**
Small patches to enable LFS

### CODE REVIEW

**Comment by SanabriaRusso** [comment]
!ci-build-me

File: buildkite/scripts/build-release.sh [hunk 1]
```diff
@@ -20,4 +20,4 @@ echo "--- Upload debs to amazon s3 repo"
 ./buildkite/scripts/publish-deb.sh
 
 echo "--- Git diff after build is complete:"
-git diff --exit-code -- .
+#git diff --exit-code -- .
```
Comment (nholland94) [inline buildkite/scripts/build-release.sh:23]: Why does this need to be removed? Is there still an environment that is missing lfs?

Comment (SanabriaRusso) [reply buildkite/scripts/build-release.sh:23]: This is the part needing fixing or a workaround. LFS leaves the local repo dirty so we need to introduce further checks to keep the test. @mrmr1993 will assist here?

**Review by nholland94 — APPROVED** [review]

**Comment by emberian** [comment]
!ci-build-me

**Comment by mrmr1993** [comment]
I've cherry-picked some parts of this to [here](https://github.com/MinaProtocol/mina/pull/15284). Closing in favour of that PR.

**Pull request closed without merging**